src/shared/loop-util.h

```diff
@@ -14,9 +14,6 @@ struct LoopDevice {
         dev_t devno;
         char *node;
         bool relinquished;
-        uint64_t diskseq; /* Block device sequence number, monothonically incremented by the kernel on create/attach, or 0 if we don't know */
```

Comment:
I'd really keep the diskseq, it's how we can pin our use of the loopback device. Even if the current dissection stuff doesn#t use it, it probably should, and condition the partition removal to only execute if it still matches the original diskseq (enclosed in a BSD lock)